internal/pkg/cli/deploy/env_test.go

```diff
@@ -293,8 +293,8 @@ func TestEnvDeployer_DeployEnvironment(t *testing.T) {
 				EnvironmentConfig: manifest.EnvironmentConfig{
 					HTTPConfig: manifest.EnvironmentHTTPConfig{
 						Public: manifest.PublicHTTPConfig{
-							SecurityGroupConfig: manifest.ALBSecurityGroupsConfig{
-								Ingress: manifest.Ingress{
+							DeprecatedSG: manifest.DeprecatedALBSecurityGroupsConfig{
```

Comment:
I thought of that but I felt that keeping old fields gives us more confidence that existing things are not breaking in any of those use cases. Because we don't want to break anything existing.